great_expectations/expectations/metrics/column_aggregate_metrics/column_median.py

```diff
@@ -27,6 +27,11 @@ class ColumnMedian(ColumnAggregateMetricProvider):
     @column_aggregate_value(engine=PandasExecutionEngine)
     def _pandas(cls, column, **kwargs):
         """Pandas Median Implementation"""
+        metrics: Dict[str, Any] = kwargs.get("metrics") or {}
```

Comment:
Could we use `get("metrics", {})` instead?

Reply:
@cdkini I believe that we could do that.  I will update in the next PR I am working on.  Thank you!